lib/entice/logic/MapInstance.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Entice.Logic.MapInstance do
+  alias Entice.Entity
+  alias Entice.Logic.MapInstance
+  alias Entice.Logic.Npc
+
+
+  defstruct(
+    players: [],
+    npcs: [], #TODO: Figure out if we want the npcs ot be able to join&leave or stay there even if dead or unspawned
```

Comment:
about the comment: i think the comment is a question we need to answer (possibly in the future?), but not here in the struct ^^